build.gradle

```diff
@@ -319,7 +334,8 @@ dependencies {
 
   testImplementation group: 'org.assertj', name: 'assertj-core', version: versions.assertJ
 
-  testImplementation group: 'com.github.hmcts', name: 'fortify-client', version: versions.fortify, classifier: 'all'
+  implementation group: 'commons-io', name: 'commons-io', version: '2.11.0'
+  testImplementation group: 'commons-io', name: 'commons-io', version: '2.11.0'
```

Comment:
(Minor) it would be nice to keep the implementation and testImplementations separate